clients/unshaded/src/main/java/tachyon/client/UfsUtils.java

```diff
@@ -93,7 +93,7 @@ private static void loadUfs(TachyonURI tfsAddrRootPath, TachyonURI ufsAddrRootPa
    * @throws IOException when an event that prevents the operation from completing is encountered
    */
   @Deprecated
-  public static void loadUnderFs(TachyonFS tfs, TachyonURI tachyonPath, TachyonURI ufsAddrRootPath,
+  public static void loadUfs(TachyonFS tfs, TachyonURI tachyonPath, TachyonURI ufsAddrRootPath,
```

Comment:
That was my hope, but as it turns out, the ``recursive`` flag of the loadFileInfoFromUfs specifies whether parent directories should be created if they do not already exist; it does not imply that contents of the path to be loaded should be loaded recursively as I have originally thought.

So unless the latter recursive behavior is implemented on the server side, we still need this class.